src/replication.c

```diff
@@ -645,11 +650,13 @@ void replicationFeedStreamFromPrimaryStream(char *buf, size_t buflen) {
     /* Debugging: this is handy to see the stream sent from primary
      * to replicas. Disabled with if(0). */
     if (0) {
```

Comment:
i think we can remove the whole if block since it is a dead code.

Reply:
Hmm, I'm generally OK leaving these because I have enabled this one specifically in the past and it's convenient to have the code easily available. Not sure if there is a better way to do this though.

Reply:
So
1. Can we just set a debug command to enable this data? IMO having things inside a null if is better be prevented.
2. In case this is only for debugging, why do we need to support the hiding feature?

Reply:
Yes, you can switch on / off this part codes through a debug command. BTW, I have no idea why these codes are here, i think it is better to leave it here based on the comments